XIPs/xip-0-purpose-process.md

```diff
@@ -38,13 +38,19 @@ An XIP must meet certain minimum criteria. It must be a clear and complete descr
 
 ## XIP Workflow
 
-### Shepherding an XIP
+## Getting started
+
+It's important to gather feedback about your proposal or new idea within the community before advancing it further in the process. This helps to vet your idea before going through the formality of the Improvement Proposal process.
+
+Engaging with the XMTP community first will help determine the originality of an idea, and will help to avoid wasting time on something that may be rejected based on prior research.
```

Comment:
engaging with the community first is important to assess demand for an idea and alignment over the solution - it's not just about originality

Reply:
@saulmc how's this?

```suggestion
Engaging with the XMTP community first will help gauge interest in the idea, determine its originality, and will help to avoid spending time on something that may be rejected based on prior research.
```